### PR TITLE
docs(migration): update content for FAQs

### PR DESCRIPTION
**Pull request by shortcuts**
## Summary

Update docs content to match the migration progress.

closes https://github.com/algolia/docsearch/pull/1288

### CODE REVIEW

File: packages/website/docs/crawler.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+You can also read our [Crawler FAQ](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq), to understand how it behaves:
+
+- [One of my pages was not crawled](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#one-of-my-pages-was-not-crawled)
+- [Why are my pages skipped?](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#when-are-pages-skipped-or-ignored)
+
+For questions related to the DocSearch program, please see our [DocSearch program FAQ](/docs/DocSearch-program).
+
+## How often will you crawl my website?
+
+Crawls are scheduled at a random time once a week, you are also able to trigger new ones directly from [the Crawler interface][2]
```
Comment (bodinsamuel) [inline packages/website/docs/crawler.mdx:19]: The schedule can be modified though

File: packages/website/docs/crawler.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Why do I have duplicate content in my results?
+
+This can happen when you have more than one URL pointing to the same content, for example with `./docs`, `./docs/` and `./docs/index.html`.
+
+We recommend configuring canonical URLs on your website, you can read more on the ["Consolidate duplicate URLs" guide by Google](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls) and use our [`ignoreCanonicalTo`](https://www.algolia.com/doc/tools/crawler/apis/configuration/ignore-canonical-to/) option directly in your crawler config.
```
Comment (bodinsamuel) [inline packages/website/docs/crawler.mdx:25]: ignoreCanonicalTo is the opposite advice of configuring canonical.
This prop is there to solves bad configuration, I would not add it to this doc

File: packages/website/docs/DocSearch-program.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+title: DocSearch program
+---
+
+If you're not finding the answer to your question on this website, this page will help you. If you're still unsure, don't hesitate to send [your question to us][1] directly.
+
+For questions related to the DocSearch x Algolia Crawler, please see our [Crawler FAQ](/docs/crawler).
+
+## What do I need to install on my side?
+
+Nothing.
```
Comment (bodinsamuel) [inline packages/website/docs/DocSearch-program.md:11]: I understand the point but it's also wrong since you need a frontend :p

**Review by bodinsamuel — APPROVED** [review]

File: packages/website/docs/crawler.mdx (hunk 1 above)
Comment (shortcuts) [reply packages/website/docs/crawler.mdx:19]: I'll add a link to the api ref

File: packages/website/docs/crawler.mdx (hunk 2 above)
Comment (shortcuts) [reply packages/website/docs/crawler.mdx:25]: Make sense, removed

File: packages/website/docs/DocSearch-program.md (hunk 3 above)
Comment (shortcuts) [reply packages/website/docs/DocSearch-program.md]: True :D Stated the frontend implementation

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**